### PR TITLE
terminal/command: 'goroutines' command add new flags '-exec command'

### LINKED ISSUE #3043: Support show friendly func args on `bt` and `goroutines -t`
Please answer the following before submitting your issue:

Note: Please include any substantial examples (debug session output,
stacktraces, etc) as linked gists.

If this is about source listing not showing up (or breakpoints not being
accepted) while running in a container please read our
[FAQ](https://github.com/go-delve/delve/blob/master/Documentation/faq.md#substpath)
first.

1. What version of Delve are you using (`dlv version`)? 1.8.3
2. What version of Go are you using? (`go version`)? go1.17.5
3. What operating system and processor architecture are you using? linux
4. What did you do? runt `bt` or `goroutines -t`
5. What did you expect to see?

show readable args on `runtime.chanrecv` and `main.chanLock.func1`:
```
0  0x000000000102e596 in runtime.gopark
   at /usr/local/Cellar/go/1.17.5/libexec/src/runtime/proc.go:367
1  0x00000000010049cc in runtime.chanrecv((*runtime.hchan)(0xc0000220c0), unsafe.Pointer(0x0), true)
   at /usr/local/Cellar/go/1.17.5/libexec/src/runtime/chan.go:576
2  0x0000000001004438 in runtime.chanrecv1()
   at /usr/local/Cellar/go/1.17.5/libexec/src/runtime/chan.go:439
3  0x00000000010574f0 in main.chanLock.func1(888)
   at ./lock/main.go:24
4  0x000000000105748d in main.chanLock·dwrap·1
   at ./lock/main.go:27
5  0x0000000001053121 in runtime.goexit
   at /usr/local/Cellar/go/1.17.5/libexec/src/runtime/asm_amd64.s:1581
```
If this feature is acceptable, I can try to contribute for it.

### PR DESCRIPTION
**Pull request by yangxikun**
Support run command on every goroutine.

Fixes #3043

### CODE REVIEW

File: pkg/terminal/command.go [hunk 1]
```diff
@@ -810,12 +816,18 @@ func printGoroutines(t *Term, indent string, gs []*api.Goroutine, fgl api.Format
 			}
 			printStack(t, t.stdout, stack, indent+"\t", false)
 		}
+		if flags&api.PrintGoroutinesExec != 0 && cmd != "" {
```
Comment (aarzilli) [inline pkg/terminal/command.go:819]: Drop the PrintGoroutinesExec flag and just check `cmd != ""`

**Review by aarzilli — COMMENTED** [review]
The documentation needs to be regenerated by running `go run _scripts/gen-cli-docs.go`

**Review by aarzilli — APPROVED** [review]
LGTM, thank you.

**Comment by derekparker** [comment]
Thanks, this will land in v1.9.1.

**Comment by derekparker** [comment]
Re-running CI.

**Comment by derekparker** [comment]
The last run had only a failure on TIP, I think this is ok to merge.

**Pull request merged**